-------------------------------------------------------------------------------
Test set: com.nnk.springboot.CurvePointTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.064 s <<< FAILURE! - in com.nnk.springboot.CurvePointTests
com.nnk.springboot.CurvePointTests  Time elapsed: 0.063 s  <<< ERROR!
org.springframework.dao.DataIntegrityViolationException: could not execute statement; SQL [n/a]; nested exception is org.hibernate.exception.DataException: could not execute statement
	at com.nnk.springboot.CurvePointTests.initCurvePoint(CurvePointTests.java:36)
Caused by: org.hibernate.exception.DataException: could not execute statement
	at com.nnk.springboot.CurvePointTests.initCurvePoint(CurvePointTests.java:36)
Caused by: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Out of range value for column 'Id' at row 1
	at com.nnk.springboot.CurvePointTests.initCurvePoint(CurvePointTests.java:36)

